ts/features/itwallet/presentation/screens/ItwPresentationCredentialDetailScreen.tsx

```diff
@@ -54,44 +82,35 @@ const ContentView = ({ eid }: { eid: StoredCredential }) => {
       />
       <ScrollView contentContainerStyle={{ paddingBottom: screenEndMargin }}>
         <View style={styles.cardContainer}>
-          <ItwCredentialCard credentialType={CredentialType.PID} />
+          <ItwCredentialCard credentialType={credential.credentialType} />
           <View
             style={[styles.cardBackdrop, { backgroundColor: themeColor }]}
           />
         </View>
 
         <ContentWrapper>
-          <ItwClaimsSections credential={eid} />
+          <ItwClaimsSections credential={credential} />
```

Comment:
I think we can remove the groups of claims that come from `ItwClaimsSections`, they are no longer needed. In the driving license detail you can still see the groups, and if you notice there is also a larger space between claims, that separates claims without any group from claims that are within a group.

The simpler solution is simply map all claims without using `groupCredentialClaims` inside `ItwClaimsSections`.

Reply:
I'm working on it in SIW-1379 :)